src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/Rfc3161TimestampTokenInfo.cs

```diff
@@ -133,6 +207,13 @@ public bool TryEncode(Span<byte> destination, out int bytesWritten)
             return true;
         }
 
+        /// <summary>
+        /// Decodes ASN.1 encoded data.
+        /// </summary>
+        /// <param name="source">Input or source buffer.</param>
+        /// <param name="timestampTokenInfo">Class representing decoded data or null when data could not be decoded.</param>
+        /// <param name="bytesConsumed">Number of bytes used for decoding.</param>
+        /// <returns>true if operation succeeded; false otherwise.</returns>
```

Comment:
```suggestion
        /// Decodes the ASN.1 encoded data.
        /// </summary>
        /// <param name="source">The input or source buffer.</param>
        /// <param name="timestampTokenInfo">When this method returns <see langword="true" />, the decoded data. When this method returns <see langword="false" />, the value is <see langword="null" />, meaning the data could not be decoded.</param>
        /// <param name="bytesConsumed">The number of bytes used for decoding.</param>
        /// <returns><see langword="true" /> if the operation succeeded; <see langword="false" /> otherwise.</returns>
```